### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by AyhanKRS** [ThreatUnkown/jsubfinder#13] [General]
I can't download wet signatures, there is no wet signature page, it gives 404 error
I will be glad if you help me, if you edit the source again, I can download it

**Comment by hiddengearz** [comment]
Thanks for spotting this. It's been resolved in 52089d161b257ef869dae7f00fa813e40298b45c

Edit: `wget https://raw.githubusercontent.com/ThreatUnkown/jsubfinder/master/.jsf_signatures.yaml && mv .jsf_signatures.yaml ~/.jsf_signatures.yaml`

**Comment by AyhanKRS** [comment]
thankyou

**Comment by AyhanKRS** [comment]
I don't understand what the outputs are saying, can you help me bro?
"Potential cryptographic key bundle .asc found in content"
"Shell configuration file .extra found in content"
"Potential cryptographic private key .pem found in content"

**Reply by hiddengearz** [reply]
a file with that extension was found in the page or JS.

There's a lot of false positives with the secrets feature

**Comment by AyhanKRS** [comment]
How should I go about finding files with this extension?
